build-scripts/for-linux/build-on-linux.bash

```diff
@@ -38,8 +38,12 @@ APP_PRMS="\
 # find swift
 APP_PRMS="$APP_PRMS -DCMAKE_Swift_COMPILER=`$BASE_DIR/find-swift.bash`"
 
-echo "env CC=clang CXX=clang++ cmake -G Ninja $APP_PRMS . $SRC_DIR"
-env CC=clang CXX=clang++ cmake -G Ninja $APP_PRMS . $SRC_DIR
+JAVA=`(readlink -f /usr/bin/javac | sed "s:/bin/javac::")`
```

Comment:
We have to find **the earliest** java version among installed ones. /usr/bin/javac may refer to any version (`update-alternative` switches one)

So please find another way of finding a proper jdk. May be ``alternatives --display java_sdk_openjdk`` is better.